README.md

```diff
@@ -52,10 +52,10 @@ This sample implements the following scenarios.
 
 To use the Microsoft Graph Webhook Sample for ASP.NET Core, you need the following:
 
-- [.NET 5.0](https://dotnet.microsoft.com/download) or later.
-- A [work, school or personal account](https://dev.office.com/devprogram). **Note:** Personal accounts cannot access the app-only Teams channel scenario in the app. A tenant administrator account is required to grant application permissions for the app-only portion.
+- [.NET 7.0](https://dotnet.microsoft.com/download) or later.

```

Comment:
```suggestion
- [.NET 8.0](https://dotnet.microsoft.com/download) or later.
```

dotnet 8 is RTM and LTS, dotnet 7 is only going to be supported until May 2024, let's jump to this one directly